lib/config.js

```diff
@@ -13,8 +13,9 @@ function Config (process) {
   this.httpProxy = process.env.IED_HTTP_PROXY || process.env.HTTP_PROXY || null
   this.httpsProxy = process.env.IED_HTTPS_PROXY || process.env.HTTPS_PROXY || null
   this.requestRetries = parseInt(process.env.IED_REQUEST_RETRIES, 10) || 5
-  this.sh = process.env.IED_SH || (process.platform === 'win32' ? process.env.comspec || 'cmd' : 'sh')
-  this.shFlag = process.env.IED_SH_FLAG || (process.platform === 'win32' ? '/d /s /c' : '-c')
+  this.sh = process.env.IED_SH || (isWindows ? process.env.comspec || 'cmd' : 'sh')
+  this.shFlag = process.env.IED_SH_FLAG || (isWindows ? '/d /s /c' : '-c')
```

Comment:
ok cool. Do you feel comfortable with this being on master or do you want to wait for #35 to be resolved?